src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.cs

```diff
@@ -118,6 +118,9 @@ public InlineCollection Inlines
 				if (_inlines == null)
 				{
 					_inlines = new InlineCollection(this);
+#if __WASM__
+					SetText(string.Empty); // To clean up the text that is set directly on the TextBlock's <p>
+#endif
```

Comment:
I tend to think it should be possible to use a runtime test to read the result of the ActualSize of the textblock, since we use the native DOM measure to get the size.

Reply:
It doesn't appear so. the ActualSize remains the same., but reading `clientSize`  through js introp does the trick.